lib/ace/mode/python_highlight_rules.js

```diff
@@ -58,7 +58,7 @@ var PythonHighlightRules = function() {
         "cmp|globals|max|reversed|zip|compile|hasattr|memoryview|round|" +
         "__import__|complex|hash|min|apply|delattr|help|next|setattr|set|" +
         "buffer|dict|hex|object|slice|coerce|dir|id|oct|sorted|intern|" +
-        "ascii|breakpoint|bytes"
+        "ascii|breakpoint|bytes|breakpoint"
```

Comment:
the breakpoint keyword is already present on this line

Reply:
I see it. Oops I guessed I missed it.

When using it on Cloud9 however, the word `breakpoint` is not highlighted.

Reply:
[c9.io](https://c9.io/)